samples/bot-task-approval/nodejs/Readme.md

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
Why you have 2 Readme.md files ??
1) samples/bot-request-approval/nodejs/Readme.md
2) samples/bot-task-approval/nodejs/Readme.md